docs: histories does not stimulate balances (#34)
rotki/docs: usage-guides/troubleshooting.md

If you still cannot find your asset, it might be ignored. By default, rotki only displays un-ignored assets in the table. To also show the ignored assets, choose the `Filter by ignored status → Show all`, and then unignore the asset manually. Additionally, you can whitelist the asset to prevent it from being automatically ignored by rotki's spam detection. (read: [whitelisting of ignored assets](/usage-guides/assets.html#whitelisting-of-ignored-assets))

If the asset is still not showing, you may need to add it manually.

> ## My balances are not showing after I populate my histories
> 
> It's not a bug. At the moment, we don't use events to stimulate current holdings. Balances are only calculated from connected exchanges, connected blockchain addresses, and manual balances. However, we have that plan in the future, to use historical events to recreate historical balances.

Reviewer: ```suggestion
## My balances are not showing after importing my history/creating history events

It's not a bug. At the moment, we don't use events to simulate current holdings. Balances are only calculated from connected exchanges, connected blockchain addresses, and manual balances.
```
